lucille-core/src/main/java/com/kmwllc/lucille/core/ConfigException.java

```diff
@@ -0,0 +1,29 @@
+package com.kmwllc.lucille.core;
+
+/**
+ * Exception for invalid configuration of a Lucille run.
+ */
+public class ConfigException extends Exception {
```

Comment:
the config library we use has `com.typesafe.config.ConfigException` and most problems with a config will be contained in that exception

does lucille need it's own Config exception separate from com.typesafe.config.ConfigException

in this PR, I only see the new ConfigException being thrown when the config file can't be found. A generic FileNotFoundException would probably suffice in that one case?

Reply:
ok so it's not being thrown when a file isn't found, rather when the config doesn't have an entry in the map
so probably what we need here is something like a RunnerManagerException for problems that can arise around starting/stopping a run

Reply:
ok changing ConfigException to RunnerManagerException